common/nwg_classes.h

```diff
@@ -13,10 +13,19 @@
 
 #include <string>
 #include <vector>
+#include <variant>
 
 #include <gtkmm.h>
 #include <glibmm/ustring.h>
 
+#ifdef HAVE_GTK_LAYER_SHELL
+#include <gtk-layer-shell/gtk-layer-shell.h>
```

Comment:
I just noticed that my draft code had this wrong. Sorry :(
pkgconfig returns `-I${includedir}/gtk-layer-shell`, i. e. this include does not use path provided by pkgconfig.
```suggestion
#include <gtk-layer-shell.h>
```

Reply:
Thanks, fixed it.